get-packages-and-swift-source.swift

```diff
@@ -212,10 +212,54 @@ if !fmd.fileExists(atPath: sdkPath) {
   try fmd.removeItem(atPath: sdkPath.appendingPathComponent("usr/share/man"))
 }
 
-_ = runCommand("patchelf", with: ["--set-rpath", "$ORIGIN",
-          "\(sdkPath.appendingPathComponent("usr/lib/libandroid-spawn.so"))",
-          "\(sdkPath.appendingPathComponent("usr/lib/libcurl.so"))",
-          "\(sdkPath.appendingPathComponent("usr/lib/libxml2.so"))"])
+let libPath = sdkPath.appendingPathComponent("usr/lib")
+/// The path to the shared object file
+func sopath(_ soname: String) -> String {
+  return libPath.appendingPathComponent(soname)
+}
```

Comment:
Do we really need a whole new function for a one-liner that is only used in three places below? I prefer if you just expanded this out below.